jax/random.py

```diff
@@ -1227,8 +1227,21 @@ def logistic(key, shape=(), dtype=onp.float64):
 
 @partial(jit, static_argnums=(1, 2))
 def _logistic(key, shape, dtype):
+  # Mathematically, we can compute the distribution by generating uniformly-distributed
+  # numbers x in the open interval (a, b) and computing:
+  #   z = log[ (x - a) / (b - x))
+  # It's important to avoid x=a or x=b, which lead to infinite values for z.
+  # The uniform() function genreates pseudorandom floating point numbers x in the
+  # semi-closed interval [0, 1), so if used directly  with (a,b)=(0,1), it will
+  # lead to infinite output in a small number of cases (as many as 1 in 2^23 for float32).
+  #
```

Comment:
Couldn't we just have written:
`logit(uniform(key, shape, dtype, minval=onp.nextafter(dtype(0), dtype(1)))`

Reply:
Maybe, but then instead of a one in 8 million chance of hitting ``-inf``, we get a one in 8 million chance of hitting ``logit(1E-45)``, which should only happen once in 10^45 if we were approximating the true distribution. But perhaps I'm overthinking it?